src/server/src/jobs/ActivationReminderJob.js

```diff
@@ -0,0 +1,85 @@
+import cron from 'node-cron';
+import Mailing from '../../emailController'
+import User from "../models/user";
+import jwt from "jsonwebtoken";
+import dotenv from "dotenv";
+dotenv.config();
+import logger from '../../logger';
+import { generateNewNotActivatedEmail } from "../resources/emails";
+
+/*
+* Returns users that didnt activate their account
+*/
+const getNewNotActivated = async () => {
+    const users = await User.find({activated: false});
+    logger.info(users);
+    return users;
+};
+
+/*
+* delete all users that didnt activate after DAYS_DELETE_UNACTIVED_USERS
+*/
+const removeOldNotActivated = async () => {
+    let now = new Date(); // get current time
+
+    //set min time
+    let min = new Date(); 
+    const gracePeriod = 24*60*60 * 1000 * process.env.DAYS_DELETE_UNACTIVED_USERS
```

Comment:
hold the calculated number and put in a comment the calculation process (the one now presented)

Reply:
fixed